### ISSUE TITLE
Issue with Ferdi tray icon, they say this issue should be reported here

### ISSUE THREAD

**Issue by fermmm** [linuxmint/cinnamon#10468]
```
 * Cinnamon version: 5.0.6
 ```

**Issue**
On Ferdi (an Electron application) both Right-click and Left-click on the tray icon opens a context menu that should be opened only with Right-click. This does not happen on Ubuntu and other OSs. I don't know if it happens on other Electron applications.
I originally reported [the issue](https://github.com/getferdi/ferdi/issues/2237) on their github page and they say it should be reported here. 

**Steps to reproduce**
1. Install Ferdi [latest version](https://github.com/getferdi/ferdi/releases) Example version: 5.6.4-nightly.7
2. Open Ferdi
3. Right click or Left click on the tray icon opens context menu
4. Do the same on Ubuntu and notice it does not happen

**Expected behaviour**
Left-click should open the Ferdi window without context menu. The context menu should only be opened with Right-click.

**Other information**
I tried booting Mint with a Live USB and the issue happens there as well.

**Comment by mtwebster** [comment]
Ferdi/electron uses AyatanaAppIndicator to display a status icon. By their definition, there *is no* activation with left-click. But there is a middle-click action, and I imagine other desktop icon handlers call that with a left-click also.

We have the same capability in cinnamon, but it's not enabled for all apps by default. There is a way for the user to add it (and maybe we can add ferdi to this list by default). The problem is, currently that's not working either for electron apps.

I have a test script for appindicator testing, and it works fine when left-click activate is enabled (it calls the middle-click action).

Maybe there's some regression in electron. This issue also affects my current slack app.

**Comment by Secret-chest** [comment]
This is a problem with Electron, as it happens in whatsapp too

**Comment by fermmm** [comment]
> edit: If you wish to add an app to the 'left-click activate' list in cinnamon, you can use dconf-editor and navigate to `org.x.apps.statusicon` and edit the `left-click-activate-apps` key. For ferdi you'd add 'ferdi' but other apps may not have such obvious names - they can 'name' their status icon whatever they'd like.

Sorry I'm not a Linux developer, you mean this issue can be fixed in Cinnamon? I added 'ferdi' on the key you mentioned, restarted system and it doesn't work. What I'm doing wrong?

![image](https://user-images.githubusercontent.com/3660373/142358528-2ec4439f-8142-44fb-90e7-5a082a1b9046.png)

**Comment by mtwebster** [comment]
No, it's broken currently, because of (presumably) an electron issue. I pointed out how to fix it if you run into this again on something else (or they fix the electron issue). Sorry for being unclear.
